example/10_conv2d_bwd_data/CMakeLists.txt

```diff
@@ -1 +1,2 @@
 add_example_executable(example_conv2d_bwd_data_xdl conv2d_bwd_data_xdl.cpp)
+target_link_libraries(example_conv2d_bwd_data_xdl PRIVATE conv_fwd_util)
```

Comment:
conv_util instead of conv_fwd_util, since it's used by backward conv too?

Reply:
You're right. I'll change it in following PR.